containers-sig-benchmarks/src/Main.hs

```diff
@@ -0,0 +1,21 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Main where
+
+import qualified Map.Contrib.Bench.Int.RO as BIRO
+import qualified Map.Contrib.Bench.Ord.RO as BORO
+import qualified Map.Contrib.Bench.Hash.RO as BHRO
+import Data.Proxy (Proxy (..))
+
+-- Map key type for benchmarks. It can be Int, Char or any
+-- type that obeys to the constraint:
+-- (Enum k, Bounded k, Key k, NFData (Map k Int))
+type K = Int
+
+main :: IO ()
+main = do
+  BIRO.benchmark "intmap" (Proxy @K)
+
+  BORO.benchmark "ordmap" (Proxy @K)
+
+  BHRO.benchmark "hashmap" (Proxy @K)
```

Comment:
I feel like these benchmarks should be grouped into `bench` function from `gauge` library as well so you can wrap everything into single `defaultMain` instead of wrapping each single benchmark.

Reply:
Reports seem to group more naturally & look less hacky this way 👍 

```bash
benchmarked intmap/lookup/all
time                 76.95 μs   (72.37 μs .. 80.65 μs)
                     0.987 R²   (0.982 R² .. 0.993 R²)
mean                 69.63 μs   (68.44 μs .. 71.05 μs)
std dev              4.557 μs   (3.534 μs .. 5.712 μs)
variance introduced by outliers: 40% (moderately inflated)

benchmarked ordmap/lookup/all
time                 151.3 μs   (147.2 μs .. 156.6 μs)
                     0.992 R²   (0.988 R² .. 0.996 R²)
mean                 155.9 μs   (153.9 μs .. 158.7 μs)
std dev              7.950 μs   (6.458 μs .. 10.01 μs)
variance introduced by outliers: 30% (moderately inflated)

benchmarked hashmap/lookup/all
time                 60.39 μs   (53.71 μs .. 65.25 μs)
                     0.955 R²   (0.936 R² .. 0.980 R²)
mean                 49.52 μs   (47.93 μs .. 51.88 μs)
std dev              5.987 μs   (4.223 μs .. 8.192 μs)
```